tools/scaffold/scripts/schema-scaffold.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+
+# URL to download Atlas Admin API Spec
+atlas_admin_api_spec="https://raw.githubusercontent.com/mongodb/atlas-sdk-go/main/openapi/atlas-api-transformed.yaml"
+
+echo "Downloading api spec"
+curl -L "$atlas_admin_api_spec" -o "./api-spec.yml"
+
+resource_name=$1
+resource_name_lower_case="$(echo "$resource_name" | awk '{print tolower($0)}')"
+resource_name_snake_case="$(echo "$resource_name" | perl -pe 's/([a-z0-9])([A-Z])/$1_\L$2/g')"
+
+cd "./internal/service/$resource_name_lower_case" || exit
```

Comment:
let's use `pushd`  and `popd` instead of `cd` so that we can see the current directory